gateleen-hook/pom.xml

```diff
@@ -25,6 +25,12 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Do you need logging in the unit tests?

Reply:
Not important. Only a matter of convenience for development.
Part of commit message:
> ... - Fix logger configuration for tests to make developers life easier.

Logs in test won't affect released artifact. Therefore why not? Are there any problems with this? How do we debug a test failing only on travis not reproducible locally for example?

Never mind. I'll remove it :)